src/utils/set-contact.js

```diff
@@ -1,30 +1,27 @@
 export default function setContact({ name, email, rsvp, dir, partstat, role, cutype, xNumGuests }) {
   let formattedParts = [];
 
+  if(typeof rsvp !== 'undefined'){
```

Comment:
What's the reason for the `typeof` check here instead of something like

```
if(rsvp !== undefined){
```

Reply:
ah yeah, I just went into old javascript habits on this, where for safety you can use typeof in case the variable hasn't been declared

```
> const { a, b } = { a: "A" }
undefined
> a !== undefined
true
> b !== undefined
false
> c !== undefined
Uncaught ReferenceError: c is not defined
> typeof c !== 'undefined'
false
> typeof c === 'undefined'
true
```

It's not necessary here though, because the deconstruction declares all the variables.